packages/modelviewer.dev/examples/tester.html

```diff
@@ -0,0 +1,227 @@
+
+    <label for="autoRotate">auto-rotate:</label>
+    <input id="autoRotate" type="checkbox"><br/><br/>
+
+    To create a seamless poster:<br/>
+    Orient model and resize window to desired size (small is recommended), then click:<br/>
+    <button onclick="createPoster()">Create Poster</button><br/>
+    Then resize window to any aspect ratio and click:<br/>
+    <button id="display" onclick="reloadScene()">Display Poster</button><br/>
+    Click the model to dismiss the poster and interact. To save your poster, click:<br/>
+    <button id="download" onclick="downloadPoster()">Download Poster</button><br/>
+    camera-orbit:
+    <div id="cameraOrbit"></div>
+    
+    </div> 
+    <model-viewer id="loading-demo" autoplay camera-controls interaction-prompt="none"
+      src="../shared-assets/models/glTF-Sample-Models/2.0/FlightHelmet/glTF/FlightHelmet.gltf" background-color="#e1e1e1" alt="A 3D model">
+    </model-viewer>
+  </div>
+
+  <div class="footer">
+    <ul>
+	  <li class="attribution">
+        <a href="https://poly.google.com/view/dLHpzNdygsg">Astronaut</a> by <a href="https://poly.google.com/user/4aEd8rQgKu2">Poly</a>,
+        licensed under <a href="https://creativecommons.org/licenses/by/2.0/">CC-BY</a>.
+      </li>
+
+      <li class="attribution">
+        <a href="https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/DamagedHelmet">Damaged Helmet</a> by <a href="https://sketchfab.com/theblueturtle_">theblueturtle_</a>,
+        licensed under <a href="https://creativecommons.org/licenses/by-nc/3.0/us/">Creative Commons Attribution-NonCommercial</a>.
+	  </li>
+	  
+	  <li class="attribution">
+		<a href="https://github.com/mrdoob/three.js/tree/dev/examples/models/gltf/RobotExpressive">RobotExpressive</a> by <a href="https://www.patreon.com/quaternius">Tomás Laulhé</a>,
+		licensed under <a href="https://creativecommons.org/publicdomain/zero/1.0/">CC0</a>.
+	  </li>
+
+      <li class="attribution">
+        <a href="https://hdrihaven.com/hdri/?h=small_hangar_01">small_hangar_01_1k.jpg</a> by <a href="https://hdrihaven.com">HDRI Haven</a>,
+        licensed under <a href="https://hdrihaven.com/p/license.php">CC0</a>.
+      </li>
+
+      <li class="attribution">
+        <a href="https://hdrihaven.com/hdri/?h=whipple_creek_regional_park_04">whipple_creek_regional_park_04_1k.hdr</a> by <a href="https://hdrihaven.com">HDRI Haven</a>,
+        licensed under <a href="https://hdrihaven.com/p/license.php">CC0</a>.
+      </li>
+    </ul>
+    <div style="margin-top:24px;" class="copyright">©Copyright 2018 Google Inc. Licensed under the Apache License 2.0.</div>
+    <div><a href="https://github.com/GoogleWebComponents/model-viewer">Github</a> ∙ <a href="https://model-viewer.glitch.me/">Glitch</a> ∙ <a href="https://github.com/GoogleWebComponents/model-viewer/issues">Bug report</a></div>
+  </div>
+
+  <script type="module" src="./built/tester.js"></script>
```

Comment:
You will need to build a "legacy" bundle for `tester.js`. Add an entry here: 

https://github.com/GoogleWebComponents/model-viewer/blob/34d8611c58e867272a780f632d719d5b39a7f1d1/packages/modelviewer.dev/scripts/create-legacy-bundles.js#L19

And a UMD bundle config like this one to `rollup.config.js`:

https://github.com/GoogleWebComponents/model-viewer/blob/600bcebf66962059bdd13ba881bea2126eed9b93/packages/modelviewer.dev/rollup.config.js#L39-L48